src/components/Intro.js

```diff
@@ -0,0 +1,206 @@
+    background: theme.palette.primary.main,
+    border: '2px solid',
+    borderColor: theme.palette.primary.main,
+    color: '#fff',
+    textAlign: 'center',
+    marginRight: theme.spacing(2),
+    fontFamily: 'Poppins',
+  },
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    marginTop: theme.spacing(6),
+    marginBottom: theme.spacing(6),
+  },
+  stepText: {
+    display: 'flex',
+    direction: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: theme.spacing(1),
+    paddingTop: theme.spacing(3),
+    paddingBottom: theme.spacing(3),
+    minHeight: theme.spacing(14),
+    [theme.breakpoints.down('sm')]: {
+      minHeight: 'unset',
+    },
+  },
+  titleTypography: {
+    marginBottom: theme.spacing(4),
+  },
+}))
+
+const Intro = ({ causeId, introData }) => {
+  const {
+    title,
+    titleImg,
+    subtitle,
+    img1Subtext,
+    img1,
+    img2Subtext,
+    img2,
+    img3Subtext,
+    img3,
+  } = introData
+  const cx = useStyles()
+  const titleImage = getImage(formatImg(titleImg))
+  const image1 = getImage(formatImg(img1))
+  const image2 = getImage(formatImg(img2))
+  const image3 = getImage(formatImg(img3))
+  return (
+    <div className={cx.wrapper}>
+      <div className={cx.title}>
+        <div className={cx.titleText}>
+          <Typography
+            className={cx.titleTypography}
+            variant="h1"
+            color="primary"
+          >
+            {title}
+          </Typography>
+          <Markdown>{subtitle}</Markdown>
+        </div>
+        <GatsbyImage
+          className={cx.titleImage}
+          image={titleImage}
+          alt={'intro title'}
+        />
+      </div>
+      <div className={cx.steps}>
+        <div className={cx.step}>
+          <div className={cx.stepText}>
+            <Avatar className={cx.numberCircle}>1</Avatar>
+            <Markdown>{img1Subtext}</Markdown>
+          </div>
+          <GatsbyImage
+            className={cx.stepImage}
+            image={image1}
+            alt={'intro step 1'}
+          />
+        </div>
+        <div className={cx.step}>
+          <div className={cx.stepText}>
+            <Avatar className={cx.numberCircle}>2</Avatar>
+            <Markdown>{img2Subtext}</Markdown>
+          </div>
+          <GatsbyImage
+            className={cx.stepImage}
+            image={image2}
+            alt={'intro step 2'}
+          />
+        </div>
+        <div className={cx.step}>
+          <div className={cx.stepText}>
+            <Avatar className={cx.numberCircle}>3</Avatar>
+            <Markdown>{img3Subtext}</Markdown>
+          </div>
+          <GatsbyImage
+            className={cx.stepImage}
+            image={image3}
+            alt={'intro step 3'}
+          />
+        </div>
+      </div>
+      <InstallButton
+        className={cx.buttonStyles}
+        color="secondary"
+        size="medium"
+        onBeforeInstall={() => {
```

Comment:
Would like some thoughts here too on how we should get the InstallButton working globally across the page even if there is no shared state across the whole application. Can we move this logic into the component? Note that I also have the BrowserUnsupported Component outside of this tag currently.

Reply:
yup we definitely can, and should move this logic to the component

Reply:
Yeah, hoist shared logic into the parent component and pass it in props to the children. In this case, it could be an `installButton` prop or maybe just an `onBeforeInstall` handler.

Reply:
i just turned it into it's own component so that we don't have to create the same logic in storybook components that wrap each section that uses install button